website/cue/reference/remap/functions/decode_snappy.cue

```diff
@@ -0,0 +1,32 @@
+package metadata
```

Comment:
There are some minor whitespaces issues:

- ` find ./website -name '*.cue' -exec cue fmt {} \;`
- ` make check-docs` to confirm

Reply:
done!